Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Plots"
 uuid = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 author = ["Tom Breloff (@tbreloff)"]
-version = "1.0.7"
+version = "1.0.8"
```

Comment:
still would have called a massive pipeline change a minor version

Reply:
Oh sorry, I was too impatient with merging then. Should I do anything about this?

Reply:
I guess it's not that important :-)
More just us getting used to the post-1.0 world